web-scraping/readme.md

```diff
@@ -0,0 +1,102 @@
+> **The goal of this guide:** You will learn how to extract information from certain web pages using [developer tools](guides/fixing-errors/readme.md), [node.js](http://blog.modulus.io/absolute-beginners-guide-to-nodejs) and [X-Ray](https://www.npmjs.com/package/x-ray).
+
+# Web scraping with [node.js](http://blog.modulus.io/absolute-beginners-guide-to-nodejs) an [x-ray](https://www.npmjs.com/package/x-ray)
+
+You can of course start here at [hackertyper](http://hackertyper.com/) and do your swordfish magic! 🤓
+
+Web scraping enables you to extract information from the web and put it in a more workable format (*.json*, *.csv*, *excel*, *google spreadsheets*).     
+
+## First things first
+
+1. Create a file ```'filename'.js``` in your designated folder. In this file you will write your code. Then, create a folder where your information will be displayed ```results.json```.
+
+* Open the [Terminal](http://www.macworld.co.uk/feature/mac-software/get-more-out-of-os-x-terminal-3608274/) application on your mac, go to the folder where you created your files by typing ```cd 'path to your folder'``` enter.
+
+* Install node.js: ```npm install node``` and enter. Then you need X-Ray: ```npm install x-ray``` enter.
+
+## Lets start scraping
+
+In this example we extract the 'experience' information from a certain LinkedIn profile. Important is that you don't log in! Or else the html code is different from this example.
+
+### Open Chromes Developer tools
+
+In the Chrome web browser open the devtools with ```cmd-alt-i``` and click on the parent element:
+
+<img src="../images/devtools-web-scraping.png" width="800">
+
+> With scraping you need to get comfortable with developer tools and with reading html code.
+
+We search 🕵 from top to bottom to find what we want:
+
+* We want all the information of experience. ```class='positions'```
+* We want the title of each experience: 'Content developer'. ```class='item-title'```.
+* We also want the organisation of each title: 'New Atoms'. ```class='item-subtitle'```.
+* finally, we want the date and time the person was active under that title and organisation: ```class='.date-range'```
+
+This ↓ is all the code we need. Not very scary, aint it?
+
+    var x = new require('x-ray')()
+
+    x(
+      'https://linkedin.com/in/some-person',
```

Comment:
I would keep this a real link, as it allows people to then copy and past the code.